05-history.md

```diff
@@ -220,3 +220,43 @@ moving backward and forward in time becomes much easier.
 >     $ git checkout <unique ID of last commit> data_cruncher.py
 >     ~~~
 > 5. Both 2 & 4
+
+
+> ## Understanding Workflow and History {.challenge}
+>
+> What is the output of the following commands?
```

Comment:
What about: "What is the output of `cat venus.txt` at the end of this set of commands?" Do you think that sounds a bit clearer?